.github/workflows/release-manual.yml

```
@@ -44,18 +44,35 @@ jobs:
       - name: Bump release version
         run: |
           echo "NEW_VERSION=$(npm --no-git-tag-version version ${RELEASE_LEVEL})" >> ${GITHUB_ENV}
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)